bin/gem_downloader

```diff
@@ -17,10 +17,27 @@ file_format = "%0#{(gp.target_files.length - 1).to_s.length}d-%s-%s.gem"
 FileUtils.remove_dir(gp.target_dir, true)
 Dir.mkdir(gp.target_dir)
 Dir.chdir(gp.target_dir) {
-  gp.target_files.each { |n, v, an|
-    path = sprintf(file_format, index, n, v)
+  gp.target_files.each { |name, version, approach|
+    path = sprintf(file_format, index, name, version)
     loop {
-      `curl -o #{path} -L http://rubygems.org/downloads/#{n}-#{v}.gem`
+      case approach
+      when :fetch
+        # Fetch the pre-compiled gem native extension instead of building them
```

Comment:
This used to be `gem native extensions` (plural). Is there a reason you changed it to singular? If yes, let's also change `them` to `it`, otherwise let's switch back to `extensions`.

Reply:
Ah, I did a full replacement which changed this one as well. Just reverted this.